extensions/wikia/AdEngine/liftium/Liftium.js

```diff
@@ -1442,19 +1496,8 @@ Liftium.init = function (callback) {
 		tracker.measureTime('adengine.init', 'liftium').track();
 	});
 
-	// TODO remove! an ugly hack for AdDriver transparency
-	var callback2 = function() {
-		if (typeof callback === 'function') {
-			callback();
-		}
-		if (window.AdEngine_loadLateAds) {
-			Liftium.d("AdEngine_run_later", 1);
-			window.AdEngine_loadLateAds();
-		}
-	};
-
 	Liftium.pullGeo();
-	Liftium.pullConfig(callback2);
+	Liftium.pullConfig(Liftium.processQueue);
```

Comment:
this method still gets callback as a parameter which is unused now. (Let's remove it).